### PR TITLE
contract documentation for Treasury

### LINKED ISSUE #541: contract documentation for Treasury

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
closes #541

### CODE REVIEW

File: main/zoe/guide/contracts/treasury.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+When borrowers exercise an invitation and deposit collateral, they receive a
+`Vault` object and some tools useful to the wallet from the offerResults. The
+`Vault` has these methods: `{ makeAdjustBalancesInvitation, makeCloseInvitation,
+getCollateralAmount, getDebtAmount, getLiquidationSeat, getLiquidationPromise
+}`.
+
+An `AdjustBalancesInvitation` allows the borrower to add or remove collateral or
+increase or decrease the loan balance.  `CloseInvitation` allows the borrower
+to close their loan and withdraw any remaining collateral.  The
+`liquidationPromise` and `liquidationSeat` allow the borrower to find out
```
Comment (katelynsills) [inline main/zoe/guide/contracts/treasury.md:32]: Should this part be updated for the recent change?

**Review by katelynsills — APPROVED** [review]
This looks like really solid material. One question about updating mention of the `liquidationPromise`.

File: main/zoe/guide/contracts/treasury.md (hunk 1 above)
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/treasury.md:32]: This part of the design ended up not being changed. The discussion we had today included some future changes, but those won't be added immediately.

Comment (katelynsills) [reply main/zoe/guide/contracts/treasury.md:32]: Hmm, is that right? The user still gets a payout through a liquidation promise that they hold? We should never have that.

Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/treasury.md]: I clarified that the `liquidationPromise` notifies the user about when liquidation happens, and the `liquidationSeat` allows them to collect the proceeds.

Comment (katelynsills) [reply main/zoe/guide/contracts/treasury.md]: Ok, sounds good!

**Pull request merged**